pkg/operation/botanist/component/vpa/general.go

```diff
@@ -17,6 +17,9 @@ package vpa
 import (
 	"fmt"
 
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
```

Comment:
Nit: You could move to the 3rd import group.

Reply:
@ialidzhikov Can we shake hands on moving the 3rd group (third-party libraries) so it becomes a 2nd group?

The insertion was made automatically by GoLand, per default configuration. I don't care how imports are organised, as long as they are sorted lexicographically. However, if I absolutely had to take a side,  I'd side with GoLand's choice: Most highly reusable imports first (standard language library). Then moderately reusable imports (domain-specific libraries). Then least reusable code - application's own peer modules.

If the team has already settled upon the other ordering as convention (stdlib -> app peer imports -> 3rd party libs), please let me know so I can configure my tooling accordingly and not fight it in the future.